module00/exercise02/main.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include <algorithm>
+#include <vector>
+
+using namespace std;
+
+bool validate_command(const std::string&);
+std::string uppercase_string(const std::string&);
+std::string lowercase_string(const std::string&);
+
+// Available commands
+const std::vector<std::string> commands = { "up", "down" };
+
+int main(int argc, char** argv)
+{
+    if(argc < 3) {
+        cout << "Example usage: ./convert {command} {string}" << endl;
+        return 1;
+    }
+
+    const string command(argv[1]);
+
+    if(!validate_command(command)) {
+        cout << "Invalid command" << endl;
+        return 1;
+    }
+
+    // Copying all arguments into a container of strings
+    vector<string> text(argv + 2, argv + argc);
+
+    // UP
+    if(command == commands[0]) {
+        transform(text.begin(), text.end(), text.begin(), ::uppercase_string);
+    }
+    // DOWN
+    else if(command == commands[1]) {
+        transform(text.begin(), text.end(), text.begin(), ::lowercase_string);
+    }
```

Comment:
```suggestion
    else {
        transform(text.begin(), text.end(), text.begin(), ::lowercase_string);
    }
```

Reply:
because you already validated the args this isn't necessary

Reply:
I added an else in case it is not of the two (in case of some option not yet implemented but that could be on the options list